crates/extensions/tedge_health_ext/Cargo.toml

```diff
@@ -0,0 +1,30 @@
+[package]
+name = "tedge_health_ext"
+description = "thin-edge Cumulocity extension adding support for health monitoring"
+version = { workspace = true }
+authors = { workspace = true }
+edition = { workspace = true }
+rust-version = { workspace = true }
+license = { workspace = true }
+homepage = { workspace = true }
+repository = { workspace = true }
+
+[dependencies]
```

Comment:
You probably need to cleanup a lot of these dependencies like `c8y_http_proxy`,  `tedge_file_system_ext` etc as many don't make much sense for this simple actor.

Reply:
Fixed